components/zcash_address/src/convert.rs

```diff
@@ -353,6 +373,17 @@ impl ToAddress for ZcashAddress {
             kind: AddressKind::P2sh(data),
         }
     }
+
+    fn from_tex(net: Network, data: [u8; 20]) -> Self {
+        ZcashAddress {
+            net: if let Network::Regtest = net {
+                Network::Test
+            } else {
+                net
+            },
```

Comment:
I'm confused as to why ZIP 320 does not specify a regtest prefix, but I don't particularly care myself

Reply:
Oh, you missed the commit that @daira added that includes regtest support.

Reply:
Done.

Reply:
Hm, I think that Daira's PR also missed this spot.